src/components/common/form/NumberInputField.tsx

```diff
@@ -18,6 +20,10 @@ export default function NumberInputField({
   const [, meta, { setValue, setError }] = useField(name)
   const decimalSeparator = (1.1).toLocaleString(i18n.lang).charAt(1)
 
+  useEffect(() => {
+    setValue(1)
```

Comment:
I'm setting the value to 1, because currently there is brief moment where the validation get's triggered and the `Минималната сума която може да дарите с карта е 1лв.` error message gets shown. This was kinda hidden by the `Collapse` animation.